### PR TITLE
VCSWP-15971: Update Python SMS Quickstart

### PR DESCRIPTION
**Pull request by lunderwood99**

### CODE REVIEW

File: Python-SMS-Quickstart.py [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
+from freeclimb.model.message_request import MessageRequest
 import os
 import json
 from flask import Flask, request
 
 configuration = freeclimb.Configuration(
-    # Defining host is optional and default to https://www.freeclimb.com/apiserver
-    host     = "https://www.freeclimb.com/apiserver",
-    # Configure HTTP basic authorization: fc
+    host = "https://www.freeclimb.com/apiserver",
```
Comment (katebrune) [inline Python-SMS-Quickstart.py]: since this is optional and external users most likely won't need to use it, can we remove?

File: README.md [hunk 2]
```diff
@@ -34,3 +34,70 @@ env FLASK_APP=python_receive_a_message_tutorial.py flask run
 ## Getting Help
 
 If you are experiencing difficulties, [contact support](https://freeclimb.com/support).
+
+# Python SMS Quickstart
```
Comment (katebrune) [inline README.md:1]: should this be at line 1?

File: README.md [hunk 3]
```diff
@@ -34,3 +34,70 @@ env FLASK_APP=python_receive_a_message_tutorial.py flask run
...
+
+A [registered application](https://docs.freeclimb.com/docs/registering-and-configuring-an-application#register-an-app) with a named alias
+
+A [configured FreeClimb number](https://docs.freeclimb.com/docs/getting-and-configuring-a-freeclimb-number) assigned to your application
+
+Trial accounts: a [verified number](https://docs.freeclimb.com/docs/using-your-trial-account#verifying-outbound-numbers)
+
+Tools:
+- [Python](https://www.python.org/downloads/) 3.5.0 or higher
+- [Yarn](https://yarnpkg.com/en/)
```
Comment (katebrune) [inline README.md]: pip

File: README.md [hunk 4]
```diff
@@ -34,3 +34,70 @@ env FLASK_APP=python_receive_a_message_tutorial.py flask run
...
+
+    ```bash
+    https://YOUR-URL.ngrok.io/incomingSms
+    ```
+    
+## Running the Quickstart
+1. Start your voice quickstart application
+
+    ```bash
+    env FLASK_APP=python_receive_a_message_tutorial.py flask run
```
Comment (katebrune) [inline README.md]: python_receive_a_message_tutorial.py -> Python-SMS-Quickstart.py

File: README.md (hunk 3 above)
Comment (lunderwood99) [reply README.md]: I think pip comes packaged with python, should I still include it?

Comment (katebrune) [reply README.md]: ya i think python has some alternative package managers so it just makes it clear we're using pip for this project

Comment (katebrune) [reply README.md]: remove yarn

**Review by katebrune — APPROVED** [review]

**Pull request merged**